webrick/upstart/tasks/RedHat.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: systemd unit - RedHat
+  template:
+    dest: "{{ webrick_systemd_unit }}"
+    src: systemd.service.j2
```

Comment:
We could add a file to `/etc/sudoers.d/` with content like this:
```
deploy ALL=(ALL) NOPASSWD: /bin/systemctl restart itedd-webrick, \
                           /bin/systemctl start itedd-webrick, \
                           /bin/systemctl stop itedd-webrick, \
                           /bin/systemctl status itedd-webrick
```

The upstart-job is sourcing `~/.profile` which in turn sets `$RAILS_ENV`, `$DATABASE_URL`,... so you can't control the environment by setting those variables before restarting. Instead you have to edit `~/.profile` for this purpose. 

So adding the above sudoer file would solve 1., 2. is anyway the same and regarding 3. there is also no  difference.  - Am I missing something?